### ISSUE TITLE
Tags newer than 3.4.0 seem to have backwards breaking changes

### ISSUE THREAD

**Issue by WillGibson** [elgalu/docker-selenium#198]
If we try to use elgalu/selenium images newer than 3.4.0 we start getting Behat exploding with messages like these...

```
Could not open connection: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'
System info: host: '9c8430f40c65', ip: '172.21.0.4', os.name: 'Linux', os.arch: 'amd64', os.version: '4.10.0-42-generic', java.version: '1.8.0_151'
Driver info: driver.version: unknown (Behat\Mink\Exception\DriverException)
```

...and...

```
Could not open connection: Unable to create a new session because of no configuration.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:40.131Z'
System info: host: '2e50ca2dfe6f', ip: '172.24.0.4', os.name: 'Linux', os.arch: 'amd64', os.version: '4.10.0-42-generic', java.version: '1.8.0_131'
Driver info: driver.version: unknown (Behat\Mink\Exception\DriverException)
```

The one above is basically the same for 3.5.3.

Maybe there is something we need to change in our Docker Compose file to work with the newer images?

This is the current working file which is pinned to elgalu/selenium:3.4.0...

```
version: '2.0'

services:
  hub:
    image: elgalu/selenium:3.4.0
    depends_on:
      - nginx
    networks:
      behat-network:
        aliases:
          - behat-browser
    volumes:
      - /dev/shm:/dev/shm
    privileged: true
    environment:
      - SELENIUM_HUB_HOST=hub
      - SELENIUM_HUB_PORT=4444
      - GRID=true
      - CHROME=false
      - FIREFOX=false
      - MOCK_SERVER_HOST=nginx
      - TZ=Europe/London

  chrome:
    image: elgalu/selenium:3.4.0
    depends_on:
       - hub
    networks:
      behat-network:
    ports:
      - 32912:25900
      - 32913:26080
    volumes:
      - /dev/shm:/dev/shm
    privileged: true
    environment:
      - NOVNC=true
      - SELENIUM_HUB_HOST=hub
      - SELENIUM_HUB_PORT=4444
      - SELENIUM_NODE_HOST={{CONTAINER_IP}}
      - SCREEN_WIDTH=1366
      - SCREEN_HEIGHT=1024
      - VIDEO=false
      - GRID=false
      - CHROME=true
      - FIREFOX=false

```

#### Operating System
Ubuntu 16.04.3 LTS

#### Image version
See versions in description above

#### Docker version
Docker version 17.06.2-ce, build cec0b72

#### Docker-Compose
docker-compose version 1.14.0, build c7bdf9e

**Comment by elgalu** [comment]
Hi, maybe you can take a look at the latest version https://github.com/elgalu/docker-selenium/blob/master/docker-compose.yml and try to spot it.

**Comment by WillGibson** [comment]
@elgalu I can't find it, I took out the `- MOCK_SERVER_HOST=nginx`, but that made no difference.

Somehow in all my upping and downing, maybe because I went to test it on a difference, simpler project, the error reported above has gone away, but I get this with elgalu/selenium:latest...

![screenshot from 2017-12-13 15-05-35](https://user-images.githubusercontent.com/8738245/33946233-234a9cea-e019-11e7-80a4-21a6e394846e.png)

So I methodically crawled my way up from 3.4.0 tag by tag and it works fine up to 3.7.1, but does like the above screenshot on 3.8.1. The only thing I changed was the elgalu/selenium tag, so maybe something has changed that's not backwards compatible.

Then I turned my attention to the original project and found the same issue as my original post above.

So far I've been unable to work out why they are behaving differently, the one that fails at 3.8.1 is a symfony project, and the other is a silverstripe project, but as I have no idea what the error means, I'm not really able to figure out where to start poking around.
